libretroshare/src/pqi/pqinetwork.cc

```diff
@@ -317,7 +322,15 @@ bool getLocalAddresses(std::list<sockaddr_storage> & addrs)
 		}
 	}
 	free(adapter_addresses);
-#else // WINDOWS_SYS
+#elif defined(__ANDROID__)
+	foreach(QHostAddress qAddr, QNetworkInterface::allAddresses())
```

Comment:
We need Qt anyway to deploy on Android, btw I have tried alternative approaches (as an example https://github.com/morristech/android-ifaddrs ) before using the one provided by Qt but I have experienced crashes, moreover we already use Qt in libretroshare if libresapi local socket server is enabled, in any case this doesn't depend on Qt by default (only if you need to deploy on android or if you need libreaspi local socket)